Example.Instrumented.Library/Format.CustomizeLogString.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace Pocket;
+
+internal partial class Format
+{
+    static partial void CustomizeLogString(object value, ref string output)
```

Comment:
Any reason to prefer `ref` instead of returning the string?

Reply:
Ah I guess this is a requirement imposed by the compiler because the method is `partial` and we don't want to force consumers to have to provide an implementation by default.